pkgs/development/compilers/ghc/7.10.3.nix

```diff
@@ -31,13 +32,21 @@
 }:
 
 let
-  inherit (bootPkgs) ghc;
```

Comment:
So e.g. this would be `inherit (bootPkgs) hscolour; bootCompiler = bootPkgs.compiler;`